lib/workers/bin/run_single_worker.rb

```diff
@@ -52,7 +56,12 @@
 worker_class = worker_class.constantize
 worker_class.before_fork
 unless options[:dry_run]
-  worker = worker_class.create_worker_record
+  worker = if options[:guid]
+             worker_class.find_by(:guid => options[:guid])
```

Comment:
haha, you can be critical of other things too... Otherwise, what's the point?  I know you're kidding but wanted to make sure you were "fine" with throwing 🍅